Backend/templates/.editorconfig

```diff
@@ -0,0 +1,20 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+end_of_line = lf
+insert_final_newline = true
+max_line_length = 120
+
+
+[*.{yaml,yml}]
+indent_size = 2
+
+
+[*.go]
+indent_style = space
```

Comment:
💄 you already have this defined for all the file types, no need to do it here.

Reply:
fixed 🤦‍♂️